extensions/amp-byside-content/0.1/amp-byside-content.js

```diff
@@ -122,8 +122,8 @@ export class AmpBysideContent extends AMP.BaseElement {
    * @override
    */
   preconnectCallback(onLayout) {
-    if (this.iframeSrc_) {
-      this.preconnect.url(this.iframeSrc_, onLayout);
+    if (this.iframeSrc_ || this.origin_) {
```

Comment:
Should be able to compose the origin here, but it'll require updating `#composeOrigin_` a bit to grab the subdomain.

Reply:
Could not get what you mean here. preconnectCallback is called after build, therefore this.origin_ is always correctly defined.